### DISCUSSION TITLE
How to add a hyperlink to text

### DISCUSSION THREAD

**Discussion by WartinCodes** [mono/SkiaSharp#2603] [Q&A] [answered]
I have SkiaSharp installed in a .NET 6 project. I have an image that I use as a template and send it by email, and I dynamically write text on it. How can I add a hyperlink to a text? How can I solve it?

**Comment by ojb500** [comment, marked as answer]
This is not really within SkiaSharp's capabilities, if you are generating a JPG/PNG image I would suggest using either [HTML image maps](https://www.emailonacid.com/blog/article/email-development/image-map-support-in-html-email/) `<map>` or [slicing your image into several smaller images](https://stackoverflow.com/a/69708033)
